src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function convertPopulationToNumber(populationString) {
+  return Number(populationString.split(',').join(''));
+}
+
+const populationElements = document.querySelectorAll('span.population');
+const populationList = [...populationElements]
+  .map(({ textContent }) => convertPopulationToNumber(textContent));
+
+const totalPopulation = populationList
+  .reduce((sum, value) => sum + value);
+const averagePopulation = totalPopulation / populationList.length;
```

Comment:
```suggestion
const averagePopulation = populationList.length
  ? totalPopulation / populationList.length
  : 0;
```

In case of empty array.